extension/helpers.js

```diff
@@ -159,12 +159,19 @@ class AES {
     
     /**
      * Cleans a string of punctuation to returns an integer
-     * @param {string} "2,000 AS$" | "2.000 AS$"
+     * @param {string} value - "2,000 AS$" | "2.000 AS$"
      * @returns {integer} 2000
      */
     static cleanInteger(value) {
+        value = value.trim()
+        const isCurrency = value.match(/\$$/g)?.length
```

Comment:
```suggestion
        const isCurrency = value.match(/\$$/g)
```

Since `.match()` returns `null` if the regex doesn't match, you can just use the return value directly. If you'd like to explicitly cast it to a boolean, you can use `Boolean(...)` or `!!(...)`.

Reply:
I didn’t know about `Boolean()`; that looks useful!